docs/glossary.md

```diff
@@ -4,41 +4,180 @@ date: 2023-11-28
 license: "This software is licensed under the Apache License version 2."
 ---
 
+## API
+
+- Application Programming Interface, defining the methods and protocols for interacting with a server.
+
+## AWS
+
+- Amazon Web Services, a cloud computing platform provided by Amazon.
+
+## AWS Glue
+
+- A compatible implementation of the Hive Metastore Service (HMS).
+
+## GPG/GnuPG
+
+- Gnu Privacy Guard or GnuPG is an open-source implementation of the OpenPGP standard.
+  It is usually used for encrypting and signing files and emails.
+
+## HDFS
+
+- **HDFS** (Hadoop Distributed File System) is an open-source distributed file system.
+  It is a key component of the Apache Hadoop ecosystem.
+  HDFS is designed as a distributed storage solution to store and process large-scale datasets.
+  It features high reliability, fault tolerance, and excellent performance.
+
+## HTTP port
+
+- The port number on which a server listens for incoming connections.
+
+## IP address
+
+- Internet Protocol address, a numerical label assigned to each device in a computer network.
+
+## JDBC
+
+- Java Database Connectivity, an API for connecting Java applications to relational databases.
+
+## JDBC URI
+
+- The JDBC connection address specified in the catalog configuration.
+  It usually includes components such as the database type, host, port, and database name.
+
+## JDK
+
+- The software development kit for the Java programming language.
+  A JDK provides tools for compiling, debugging, and running Java applications.
+
+## JMX 
+
+- Java Management Extensions provides tools for managing and monitoring Java applications.
+
+## JSON
+
+- JavaScript Object Notation, a lightweight data interchange format.
+
+## JSON Web Token
+
+- See [JWT](#jwt).
+
+## JVM
+
+- A virtual machine that enables a computer to run Java applications.
+  A JVM implements an abstract machine that is different from the underlying hardware.
+
+## JVM instrumentation 
+
+- The process of adding monitoring and management capabilities to the [JVM](#jvm).
+  The purpose of instrumentation is mostly for the collection of performance metrics.
```

Comment:
Suggestion  = "The purpose of instrumentation is (/mainly) to collect performance metrics."